spec/models/trip_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe Trip, type: :model do
+  describe 'Validations' do
```

Comment:
@jrambold do know if the trip data is clean? Should we validate all the attributes or do we want to allow for incomplete records?